### PR TITLE
Changed to only save cookie once as multiple calls were replacing pre…

### PR DESCRIPTION
**Pull request by MichaelYoung1981**
…vious updates.

### CODE REVIEW

File: src/SFA.DAS.CloudService/SFA.DAS.EAS.WebContent/diagnostics.wadcfgx [hunk 1]
```diff
@@ -43,5 +43,5 @@
   <PrivateConfig>
     <StorageAccount name="" key="" endpoint="" />
   </PrivateConfig>
-  <IsEnabled>true</IsEnabled>
+  <IsEnabled>false</IsEnabled>
```
Comment (tom-gough) [inline src/SFA.DAS.CloudService/SFA.DAS.EAS.WebContent/diagnostics.wadcfgx]: what is the motivation behind this change?

Comment (MichaelYoung1981) [reply src/SFA.DAS.CloudService/SFA.DAS.EAS.WebContent/diagnostics.wadcfgx]: Dev Ops removed a load of the diagnostics config so the app wouldn't run locally without making that change.

File: src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CookieBasedOrganisationAndPayeRefDataSaver/Given_Cookie_Does_Not_Already_Exist.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                        act: f => f.Handle(),
+                        assert: f =>
+                            f.CookieRepository
+                                .Verify(
+                                    m =>
+                                        m.Update(
+                                            It.IsAny<string>(),
+                                            It.IsAny<EmployerAccountData>()),
+                                    Times.Never));
+            }
```
Comment (tom-gough) [inline src/SFA.DAS.EmployerAccounts.UnitTests/Commands/CookieBasedOrganisationAndPayeRefDataSaver/Given_Cookie_Does_Not_Already_Exist.cs]: indentation looks slightly off here

File: src/SFA.DAS.EmployerAccounts/Commands/OrganisationAndPayeRefData/CookieBasedOrganisationAndPayeDataSaver.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.Account;
+
+namespace SFA.DAS.EmployerAccounts.Commands.OrganisationAndPayeRefData
+{
+    public sealed  class CookieBasedOrganisationAndPayeDataSaver : AsyncRequestHandler<SaveOrganisationAndPayeData>
```
Comment (tom-gough) [inline src/SFA.DAS.EmployerAccounts/Commands/OrganisationAndPayeRefData/CookieBasedOrganisationAndPayeDataSaver.cs]: why not handler?

File: src/SFA.DAS.CloudService/SFA.DAS.EAS.WebContent/diagnostics.wadcfgx (hunk 1 above)
Comment (tom-gough) [inline src/SFA.DAS.CloudService/SFA.DAS.EAS.WebContent/diagnostics.wadcfgx]: what is the motivation behind this change?

File: src/SFA.DAS.EmployerAccounts/Commands/OrganisationAndPayeRefData/CookieBasedOrganisationAndPayeDataSaver.cs (hunk 3 above)
Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerAccounts/Commands/OrganisationAndPayeRefData/CookieBasedOrganisationAndPayeDataSaver.cs]: I was following the existing naming convention of the command I'd copied.  I did think about changing it but it would have then been inconsistent.
